server/routes/filmsRoutes.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Film = mongoose.model('films');
+const User = mongoose.model('users');
+
+module.exports = (app) => {
+  app.get(`/api/films/:pageId/`, async (req, res) => {
+    const { pageId } = req.params
+    i = 5
+    y = i * (pageId - 1)
+    const films = await Film.find().skip(y).limit(i);
+
+    return res.status(200).send(films);
+  });
+
+  app.get(`/api/film/title`, async (req, res) => {
+    const films = await Film.find({ "title": { "$regex": req.query.keyword }});
+
+    return res.status(200).send({
+      error: false,
+      films,
+    });
+  });
+
+  app.get(`/api/film/actor`, async (req, res) => {
```

Comment:
This looks almost exactly the same as 
[`/api/film/title`](https://github.com/jbryant92/black-codher-personal-project/pull/1/files#diff-667455c95b33412ef6c29b2f3d6ce1b342cc8af89f918006b885d244bb4249f8R15), I think we could just make `/film` and `/title` a parameter for the lookup.

Alternatively we could move the duplicate code into a function call that will be shared by both